lib/typescript/httpclient/model/Message.ts

```diff
@@ -47,6 +47,12 @@ export enum SenderType {
   sourceUser = 'source_user',
   appUser = 'app_user',
 }
+
+export function isFromContact(message: Message) {
```

Comment:
Let's try and keep all methods that only operate on the content model right in the `model/` folder :) This way we can easily replace these methods if the logic changes and don't have go though UI code to change it.